setup.cfg

```diff
@@ -17,7 +17,7 @@ classifiers =
     License :: OSI Approved :: MIT License
 
 [options]
-zip_safe = True
+zip_safe = False
```

Comment:
Would be nice to mention https://mypy.readthedocs.io/en/stable/installed_packages.html#making-pep-561-compatible-packages

Reply:
Done